### ISSUE TITLE
Verification measurement report is not adjusting to the simulation profile

### ISSUE THREAD

**Issue by 3ll3d00d** [eoyilmaz/displaycal-py3#276]
this appears to be a regression vs 3.8.9.3 

using these settings

on 3.8.9.3
![verification_3893](https://github.com/eoyilmaz/displaycal-py3/assets/2487509/dd12d25e-23c0-4cbb-9e40-8fa74c214cca)

on develop (same for published version)
![verification_develop](https://github.com/eoyilmaz/displaycal-py3/assets/2487509/44c0efab-d069-4704-9e47-bb86d6515a2b)

produces this report on 3.8

![verification_report_3893](https://github.com/eoyilmaz/displaycal-py3/assets/2487509/388e60ee-ee82-4281-bd86-3d0a0bc1e02e)

but this one on develop

![verification_report_develop](https://github.com/eoyilmaz/displaycal-py3/assets/2487509/5a4d9d77-c5e2-4782-91ef-495184d09cd1)

the nominal gamma is not adjusted which leads to large reported errors

**Comment by 3ll3d00d** [comment]
also note how the output profile field is now always visible on develop

**Comment by 3ll3d00d** [comment]
`output_profile_ctrl` is declared in `report.xrc`

```
                <object class="FileBrowseButtonWithHistory" name="output_profile_ctrl">
                  <toolTip>output.profile</toolTip>
                  <hidden>1</hidden>
                </object>
```

on the 3.8 code, `FileBrowseButtonXmlHandler`  creates the above

after this call

https://github.com/eoyilmaz/displaycal-py3/blob/main/DisplayCAL/xh_filebrowsebutton.py#L45

`w.Shown` becomes False

on develop, it stays on True

so seems to be a regression somewhere in the displaycal specific wx code?

**Comment by 3ll3d00d** [comment]
same problem applies to `input_value_clipping_bmp`

**Comment by 3ll3d00d** [comment]
fixed proposed in #278
